api/tests/Behat/features-v2/reporting/sections/debts/debts.feature

```diff
@@ -0,0 +1,32 @@
+@v2 @debts
+Feature: Report debts
+
```

Comment:
Yeah you sort of don't have an option to add a new debt from the summary page. Once you have selected yes when stating whether the client has a debt or not you are taken to the debts edit page where you have to put a value inside at least one debt (Care fees, Credit cards etc.).

Once this is done there is no option to add another debt from the debts summary page instead the only option is to edit the current list of debts and update a value. I believe this is because the type of debts are not optional and instead predefined as care fees, credit cards, loans, or other.

The same for removing a debt you can't really remove a debt all you can do is update existing debt to put a debt value to £0

Hopefully this clears that up.

Reply:
Yep thats great